### PR TITLE
feat(frontend):  sign-up-mobile-responsiveness bb-60

### PR DESCRIPTION
**Pull request by iqbalalayubbi**
**responsive start from under 641 px**

### CODE REVIEW

File: apps/frontend/src/pages/auth/style.module.css
[the remark this answers is not in the document]
Comment (JD-Rosales) [reply apps/frontend/src/pages/auth/style.module.css:1]: I also noticed that you force pushed without properly resolving the merge conflict and might lead to issues.

File: packages/shared/CHANGELOG.md [hunk 1]
```diff
@@ -22,26 +22,68 @@
 * authorization token (JWT) bb-10 ([#40](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/issues/40)) ([16e3c35](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/commit/16e3c353ff700ab27a7b6af4fa7b3c17059cc916))
 
 ## [1.2.0](https://github.com/BinaryStudioAcademy/bsa-2024-bebalance/compare/shared-v1.1.0...shared-v1.2.0) (2024-08-21)
+>>>>>>> f9749449ad8a29e69c8f5599fce32330826a92fd
```
Comment (yulyaolshanska) [inline packages/shared/CHANGELOG.md:25]: I think it is left over from the resolution of the conflict? And I think we should not modify this file.

File: apps/frontend/src/libs/components/link/link.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { type AppRoute } from "~/libs/enums/enums.js";
 import { getValidClassNames } from "~/libs/helpers/helpers.js";
 import { type ValueOf } from "~/libs/types/types.js";
 
-import { type LinkType } from "./libs/types/types.js";
+import { LinkType } from "./libs/types/types.js";
```
Comment (yulyaolshanska) [inline apps/frontend/src/libs/components/link/link.tsx:7]: Why you remove type?

File: apps/frontend/src/pages/auth/style.module.css [hunk 3]
```diff
@@ -25,7 +25,7 @@
 	color: white;
 }
 
-.illustration-container .title {
+.ilustration-container .title {
```
Comment (yulyaolshanska) [inline apps/frontend/src/pages/auth/style.module.css:28]: Typo in "ilustration"

File: apps/frontend/src/pages/auth/style.module.css
[the remark this answers is not in the document]
Comment (iqbalalayubbi) [reply apps/frontend/src/pages/auth/style.module.css:1]: Okey, i try to fix this thank you

File: apps/frontend/src/libs/components/link/link.tsx (hunk 2 above)
Comment (iqbalalayubbi) [reply apps/frontend/src/libs/components/link/link.tsx:7]: i'm forget about this. thank you

File: apps/frontend/src/pages/auth/assets/icons/eye-slash.svg
Comment (JD-Rosales) [inline apps/frontend/src/pages/auth/assets/icons/eye-slash.svg:1]: Asset files should be at ~/assets/

**Review by JD-Rosales — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/assets/icons/eye-slash.svg
Comment (fshabanov) [inline apps/frontend/src/pages/auth/assets/icons/eye-slash.svg:1]: These icons shouldn't have been added

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx [hunk 4]
```diff
@@ -29,64 +28,46 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+			</span>
+			<form onSubmit={handleFormSubmit}>
+				<div className={styles["input-groups"]}>
 					<Input
 						control={control}
 						errors={errors}
 						label="Email"
 						name="email"
-						placeholder="name@gmail.com"
+						placeholder="email"
```
Comment (ihorLysak) [inline apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx:53]: placeholder should be email@example.com

**Review by fshabanov — CHANGES_REQUESTED** [review]
Your PR shouldn't have any CHANGELOG chages.
Also your PR says mobile responsiveness for sign up, but file changes are for sign in

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 5]
```diff
@@ -58,21 +58,42 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 						errors={errors}
 						label="Email"
 						name="email"
-						placeholder="email"
+						placeholder="name@gmail.com"
```
Comment (ihorLysak) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:61]: Sould be "name@example.com"

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 6]
```diff
@@ -58,21 +58,42 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+					<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
+				</form>
+			</div>
+
+			<div className={styles["logo-container"]}>
+				<span className={styles["logo"]}>LOGO</span>
+			</div>
+
+			<img
+				alt="ripple-effect-bg"
```
Comment (ihorLysak) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:82]: alt should have a meaningful string isnisde of them, like "ripple visual effect"

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 7]
```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+		<div className={styles["container"]}>
+			<div className={styles["form-container"]}>
+				<form className={styles["form"]} onSubmit={handleFormSubmit}>
+					<div className={styles["form-header"]}>
+						<div className={styles["form-header__logo-container"]}>
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>LOGO</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>CREATE AN ACCOUNT</h1>
```
Comment (ihorLysak) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:41]: On my PR, I was instructed to use 	`text-transform: uppercase;` in CSS for cases like this one. Using this rule, you can provide a lowercase text in JSX and CSS will transform it to upper case.

Comment (iqbalalayubbi) [reply apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:41]: but, i see in the repo now used upper case text directly.

Comment (fshabanov) [reply apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:41]: We should update to `text-transform: uppercase`

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 8]
```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+						<span className={styles["form-header__sub-text"]}>
+							Already have an account? Go to{" "}
+							<Link to={AppRoute.SIGN_IN}>Sign In</Link>
+						</span>
+					</div>
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:46]: Shouldn't be within `<form>` tag

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx (hunk 5 above)
Comment (fshabanov) [reply apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:60]: Not fixed

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by yulyaolshanska — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Pull request merged**